roles/confluent.variables/vars/main.yml

```
@@ -740,8 +844,37 @@ control_center_properties:
     properties:
       confluent.controlcenter.rest.authentication.method: BEARER
       public.key.path: "{{rbac_enabled_public_pem_path}}"
-      confluent.metadata.bootstrap.server.urls: "{{mds_urls}}"
+      confluent.metadata.bootstrap.server.urls: "{{mds_bootstrap_server_urls}}"
       confluent.metadata.basic.auth.user.info: "{{ control_center_ldap_user | default('c3') }}:{{ control_center_ldap_password | default('pass') }}"
+  rbac_external_client:
+    enabled: "{{rbac_enabled and external_mds_enabled and mds_broker_listener.ssl_enabled }}"
```

```suggestion
    enabled: "{{rbac_enabled and external_mds_enabled and mds_ssl_enabled }}"
```
please check on the below properties, most are to connect to other systems than the mds server, also the mds server only has 1 way tls on